servo-gecko-try/init.sls

```diff
@@ -0,0 +1,91 @@
+{% from tpldir ~ '/map.jinja' import servo-gecko-try %}
+
+user-acct:
+  user.present:
+    - name: servo-gecko-try
+    - fullname: Gecko T. Robot
+    - shell: /bin/bash
+    - home: /home/servo-gecko-try/
+
+packages:
+  pkg.installed:
+    - names:
```

Comment:
Use `pkgs` instead of `names`, per the [style guide](https://github.com/servo/saltfs/blob/master/STYLE_GUIDE.md). This can also get moved under `servo-gecko-try`.